src/data/help.csv

```diff
@@ -0,0 +1,83 @@
+"開催したレースは参加リストから参加者を募集できます。\n<color=#ff9900>サークル共有機能</color>や<color=#ff9900>SNS共有機能</color>を使って、\n<color=#ff9900>レースID</color>を入力することで身近なトレーナーも募集できます。\nまた、お気に入りのレースを見返せる<color=#ff9900>保存機能</color>もご活用下さい。\n友人たちと一緒にレースを開催してみるのもオススメです。","You can recruit participants for the races you have held from the participation list You can also recruit trainers close to you by entering their <color=#ff9900>race IDs</color> using the <color=#ff9900>Circle Sharing and SNS Sharing functions</color> You can also <color=#ff9900>save your favorite races</color> for future reference We recommend organizing a race with your friends!"
+"特別移籍は殿堂入りウマ娘を<color=#ff9901>他のチームに移籍させることで\nお礼品をGET</color>できます。\nなお、他のチームに送り出した殿堂入りウマ娘はいなくなり、\n元に戻すことはできませんのでご注意ください。","In the special transfer, you can transfer your Hall of Famer to another team and get a reward. Please note that once you send a Hall of Famer to another team, she will be gone and you will not be able to get her back."
+"移籍先チームの希望する条件に応じて<color=#ff9901>お礼品の内容が変わります。</color>\nまた、それぞれの募集には期待度があり、<color=#ff9901>期待度が高いほど\n殿堂入りウマ娘を送り出してGETできるお礼品は豪華</color>になります。","The contents of the <color=#ff9901>rewards will change depending on the desired conditions of the team you are transferring to. </color> In addition, each recruitment has its own expectation level, and the <color=#ff9901>higher the expectation level, the more luxurious the reward that you can get by sending Uma Musume into the Hall of Fame</color>."
+"期待度の高い募集の中にはお礼品として\n移籍した<color=#ff9901>殿堂入りウマ娘のピース</color>が手に入るものもあります。","Some of the most highly anticipated recruits will get a piece of the transferred <color=#ff9901>Hall of Fame horse girl</color> as a thank you gift."
+"チームメンバーに特殊なアイコンが出ているトレーニング場所で\nトレーニングを行うと「<color=#ff9900>アオハル特訓</color>」が発生し、\n<color=#ff9900>そのチームメンバーの基礎能力が上がります</color>。\nアオハル杯で優勝できるように、\nアオハル特訓を駆使してチームを成長させましょう！","If you train at a training location where a team member has a special icon, \"<color=#ff9900>Aoharu Special Training</color>\" will occur, and <color=#ff9900>that team member's basic abilities will increase</color>.\nLet's use the Aoharu Training to grow our team so we can win the Aoharu Hai!"
+"さらに、<color=#ff9900>2人以上</color>のチームメンバーとアオハル特訓を発生させると\n<color=#ff9900>育成ウマ娘のトレーニングの効果もアップ</color>！\nまた、特訓に参加した<color=#ff9900>チームメンバーの数が増えるほど</color>、\n<color=#ff9900>トレーニングで得られる基礎能力やスキルPtが大きくなります</color>。\nただし、その分<color=#ff9900>消費する体力も大きくなる</color>ため注意しましょう。","In addition, a special Aoharu training session <color=#ff9900>with 2 or more</color> team members, will <color=#ff9900>boost your Uma Musume's training efficiency as well Also, <color=#ff9900>the more team members</color> participate in the training, <color=#ff9900>the more basic abilities and skill points you will gain</color> from the training However, be aware that <color=#ff9900>the amount of energy you consume will also increase</color>."
```

Comment:
this is bad lol, pls use size markup and remove 1 or 2 color markups
(https://user-images.githubusercontent.com/80930594/132095541-40f123a4-8ca1-41fb-a9f0-70a7d73460bf.png)